api/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "api",
+  "private": true,
+  "version": "0.0.0",
+  "description": "API for the zeit/now repo",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc",
+    "now-build": "yarn --cwd .. && node ../run.js build all && tsc"
```

Comment:
I'm not sure if you need `tsc` here since that will already happen for `@now/node`

Reply:
yes, looks like it works without it 👍